Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockSupport.cs

```diff
@@ -23,13 +23,17 @@ public static void Initialize()
             #endif
 
             #if UNITY_EDITOR || UNITY_PS4
+
             InputSystem.RegisterControlLayout<PS4TouchControl>("PS4Touch");
             InputSystem.RegisterControlLayout<DualShockGamepadPS4>("PS4DualShockGamepad",
                 deviceDescription: new InputDeviceDescription
             {
                 deviceClass = "PS4DualShockGamepad",
                 interfaceName = "PS4"
             });
+
+            InputSystem.onDeviceChange += DualShockGamepadPS4.OnDeviceChange;
```

Comment:
This should not be necessary but the current API is indeed lacking a good way to do this. `MakeCurrent` only covers a subset of the functionality necessary to support this. I'll take a look after the PR is in.